releasenotes/notes/log-manager-18fb90743f423087.yaml

```diff
@@ -0,0 +1,6 @@
+features:
+  - |
+    Log manager for log caching
+    When Restraint runs under Beaker, harness and task logs are cached
+    in the system. Logs are uploaded to Beaker after the task completes.
```

Comment:
Perhaps add something about............
This allows restraint commands rstrnt-report-result and rstrnt-report-log to freely upload reports/logs with considerably less delay.  When harness.log and task.log files are finally uploaded, they are sent up more efficiently in a batch  instead of line-by-line.  During task execution these log files are written to disk, freeing up restraint memory usage.

Reply:
I'm not adding implementation details in the release notes.